src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -674,6 +674,45 @@ public function unitAction(Request $request, $action, $id)
         }
     }
 
+    public function exportAction($_format)
+    {
+        // init vars
+        $productProvider = $this->container->get('prestashop.core.admin.data_provider.product_interface');
+        /* @var $productProvider ProductInterfaceProvider */
```

Comment:
I found this sample on phpdocumentor docs (http://manual.phpdoc.org/HTMLSmartyConverter/PHP/phpDocumentor/tutorial_tags.var.pkg.html), still not official docs to me :)